config/initializers/devise.rb

```diff
@@ -306,9 +306,21 @@ def i18n_message(default = nil)
   config.sign_out_via = :get
 
   # ==> OmniAuth
-  # Add a new OmniAuth provider. Check the wiki for more information on setting
-  # up on your models and hooks.
-  # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :openid_connect, {
+    name: :gov_one,
+    scope: %i[openid email],
+    response_type: :code,
+    client_options: {
+      port: 443,
+      scheme: 'https',
+      host: 'oidc.integration.account.gov.uk',
```

Comment:
Should this be GOV_ONE_BASE_URI ?

Reply:
Yes, thanks [7294ab7](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/7294ab73601bba2594c50ed4a4e2e664298b9593)